<html>
  <head>
    <title>Step 1 - Input Data</title>
  </head>
  <body>
    <h1>Input Data</h1>

    <p>
      Profiling data and domain knowledge are specified in this step.
    </p>

    <ul>
      <li>
        <b>Gene list</b> - A text file containing a list of nodes
        (genes, proteins) involved in the study. One node per line. No
        row/column names.
      </li>
      <li>
        <b>Data file 1</b> - Profiling data (gene expression, protein
        expression, RNA-seq RPKM, etc.) of condition 1 in a text file
	with no row/column names. One node per row, corresponding 
	to the same order in gene list file. Tab or comma delimited. 
	Data should be properly normalized.
      </li>
      <li>
        <b>Data file 2</b> - Profiling data (gene expression, protein
        expression, RNA-seq RPKM, etc.) of condition 2 in a text file
	with no row/column names. Not required or ignored when
	experiment condition is set to single condition.
      </li>
      <li>
        <b>Prior knowledge network</b> - Prior knowledge of network topology in KGML format or a text file with no row or column names. A KGML xml file can be downloaded from the KEGG website (http: //www.genome.jp/kegg). Text file input supports either general prior knowledge applied to both conditions or condition-specific prior knowledge. Text file input has no row or column names, and a row is a record of prior knowledge edge. If a row consists of two nodes names separated by tab or comma, it is identified as general prior knowledge. If a row consists of two nodes and a value of 1 or 2 separated by tab or comma, it is identified as a condition-specific prior knowledge that is only for condition 1 or 2. This file is optional. If provided, KDDN will use the knowledge in learning, otherwise it relies solely on data.
      </li>
    </ul>

    <p>
      File choices can be canceled using button.
    </p>

</body>
</html>
